triage/storage.py

```diff
@@ -128,6 +128,21 @@ def labels(self):
     def uuid(self):
         return self.metadata['metta-uuid']
 
+    def columns(self, include_label=False):
+        columns = self.matrix.columns.tolist()
+        if include_label:
+            return columns
+        else:
+            return [
+                col for col in columns
+                if col != self.metadata['label_name']
+            ]
+
+    def matrix_with_sorted_columns(self, columns):
+        if self.columns() != columns:
+            logging.warning('Column orders not the same, re-ordering')
+        return self.matrix[columns]
```

Comment:
In the case where the `columns` parameter includes a column not in the matrix, this will throw an exception, but in the case where `columns` includes a subset, this will return a subset of the matrix, which probably isn't what we want -- I think we might want to throw an exception in both cases, right?